service/frontend/workflowHandler.go

```diff
@@ -3882,9 +3876,12 @@ func (wh *WorkflowHandler) getHistory(
 				tag.WorkflowID(execution.GetWorkflowId()),
 				tag.WorkflowRunID(execution.GetRunId()),
 				tag.Error(err))
+
+			// REVIEW: I expected this line to be here but it isn't. Should it be?
+			// return nil, nil, err
```

Comment:
Was surprised that this error gets swallowed but I didn't want to add the return line without understading if this is intentional